### ISSUE TITLE
Only generate pretty diffs for new/updated symbols

### ISSUE THREAD

**Issue by davidrpoliti** [mathieudutour/git-sketch-plugin#149]
Context: Right now, the plugin generates pretty diffs for every symbol. Repo maintainers are only interested in changes; there's no value in GIT generating pretty diffs for 500+ unchanged symbols.

Request: include a preference to only generate pretty diffs on changed symbols

**Comment by davidrpoliti** [comment]
Does Kaktus fix this by doing the diffs in the app and not committing unchanged files?

**Comment by mathieudutour** [comment]
Yes, Kactus looks at the actual changes and doesn’t make you commit an image. The image is recreated from the changes
